src/aat/resources/features/F-1023 - Case update with caseAccessGroups/S-1023.1.td.json

```diff
@@ -56,32 +56,35 @@
       "last_state_modified_date": "[[ANYTHING_PRESENT]]",
       "security_classification": "PUBLIC",
       "case_data": {
-        "CaseAccessGroups" : [ {
-          "value" : {
-            "caseAccessGroupType" : "CCD:all-cases-access",
-            "caseAccessGroupId" : "test"
-          },
-          "id" : null
-        } ]
+        "CaseAccessGroups": []
```

Comment:
Helen, why have you removed CaseAccessGroups from the Case_Data. This will test the removal of "caseAccessGroupType" = "CCD:all-cases-access".

Reply:
This is the expected response.
It is not currently returning anything back, should it?  If so something is not working correctly